anonymizer/plugin.json

```diff
@@ -1,9 +1,9 @@
 {
     "id": "anonymizer",
-    "version": "0.0.4",
+    "version": "0.0.5",
     "meta": {
-        "label": "Data anonymizer",
-        "description": "Preparation processor for anonymizing data",
+        "label": "Data anonymizer (deprecated)",
```

Comment:
question (non blocking) : should we also specify that the plugin is deprecated in the help section of the processor (method getHelp) ?

Reply:
Good point. @chaokang worked on a feature to provide deprecation/replacement for processor. I don't know if we want to invest these effort nor if the system is compatible with plugins  (I could give a try)

Reply:
done in 3ee196ff12956dd9e8286c689b16215680ec6974